cmd/bisync/cmd.go

```diff
@@ -142,7 +144,8 @@ func init() {
 	flags.BoolVarP(cmdFlags, &Opt.IgnoreListingChecksum, "ignore-listing-checksum", "", Opt.IgnoreListingChecksum, "Do not use checksums for listings (add --ignore-checksum to additionally skip post-copy checksum checks)", "")
 	flags.BoolVarP(cmdFlags, &Opt.Resilient, "resilient", "", Opt.Resilient, "Allow future runs to retry after certain less-serious errors, instead of requiring --resync. Use at your own risk!", "")
 	flags.BoolVarP(cmdFlags, &Opt.Recover, "recover", "", Opt.Recover, "Automatically recover from interruptions without requiring --resync.", "")
-	flags.IntVarP(cmdFlags, &Opt.Retries, "retries", "", Opt.Retries, "Retry operations this many times if they fail", "")
+	flags.IntVarP(cmdFlags, &Opt.Retries, "retries", "", Opt.Retries, "Retry operations this many times if they fail (requires --resilient).", "")
```

Comment:
These are overriding the global `--retries` and `--retries-sleep` flags which I didn't know cobra could cope with! Is it necessary to override them? You could just be using the values in the global config?

Reply:
This was the first thing I tried! The problem is that they don't seem to be in the global config... instead they are binding to private variables in [`cmd`](https://github.com/rclone/rclone/blob/91b54aafccc3aed01a5389864e8ef3902286b406/cmd/cmd.go#L55-L56) (unlike [`LowLevelRetries`](https://github.com/rclone/rclone/blob/91b54aafccc3aed01a5389864e8ef3902286b406/fs/config.go#L77) which is in the config.) So I couldn't find an easy way to reference them without changing too much. I found `pflag.CommandLine.Lookup("retries-sleep").Value` but that only wanted to give me a `string`, and `pflag.CommandLine.GetDuration` also didn't work because the value is `*fs.Duration`. Is there an easier way to do this? 

Alternatively, what would you think about making the variables exported or moving them to the config? I think that would be ideal, and then bisync would not need duplicate flags 🙂

Reply:
Yes making the flags global seems like the sensible thing to do, though you may have troubles with that since cmd is right at the top of the include tree. Give it a go - it it works, go for it!